src/content/reference/cluster-upgrades/index.md

```diff
@@ -0,0 +1,190 @@
+
+### Make rescheduling graceful
+
+To make rescheduling during upgrades of both the cluster as well as your own deployments more graceful you should take care of a few settings on your Pods. However, these settings might not be enough, your application must be able to handle standard termination signals and have a procedure for gracefully shutting down. Further, it needs to expose some sort of liveness and/or readiness endpoints for Kubernetes to be able to probe it.
+
+1. Have well implemented [liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/).
+   Often, a container being alive does not mean it is ready to recieve traffic. By differniating between liveness and readiness you can not only control when traffic gets routed to a fresh container, you can also influence graceful termination of your container.
+2. Set proper values for initialDelaySeconds (related to previous point).
+3. Set proper values for termincationGracePeriod to give your Pod time to gracefully shut down.
+
+### Set scheduling priorities
+
+Consider using Pod priority to ensure that higher priority Pods are scheduled favorably in times of resource pressure.
+
+We recommend reading the upstream documentation abour [priority classes and pod preemption](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/) to get a better understanding of how the scheduler works with these.
+
+To help the scheduler further with being able to correctly (re-)schedule your Pods, you should [set resource request and limits](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/). This also sets the Quality of Service of a Pod, which again has influence on scheduling priorities.
+
+### Avoid ephemeral resources
+
+In Kubernetes it is possible to schedule ephemeral resources.
+
+For example a Pod by itself, i.e. without a deployment, daemonset, statefulset, etc., is an ephemeral resource that will not be rescheduled when it dies or gets killed. A Pod definition should thus only be used for use cases like debugging or quick manual tests. Be sure to always use a controller-managed resource for your containers.
+
+Furthermore, local storage in form of `emptyDir` is also ephemeral, and should not be used to persist data. It should only be used as a cache or temporary storage that you can live without in case of failure or rescheduling. In most cases this is also true of `hostPath` as the local storage of a new node might not be the same as the one of the old node.
+
+### General Container Hygene
+
+There's additional general container hygene recommendations that will help smoothen the upgrade process.
+
+As container images might not be already available on the new node that the Pod gets rescheduled to you should make sure that all container images (and tags) that you are using are available in the registry that is configured for the Pods.
```

Comment:
`that is configured` -> `that are configured`

Reply:
the registry that is, not are